packages/peregrine/lib/store/actions/checkout/asyncActions.js

```diff
@@ -38,7 +43,7 @@ export const cancelCheckout = () =>
 export const resetCheckout = () =>
     async function thunk(dispatch) {
         await dispatch(closeDrawer());
-        await dispatch(createCart());
+        await dispatch(removeCart());
```

Comment:
This moves the creation of the cart down the line to the next action that depends on the cart id. Technically it's a tiny bit slower than how it behaved before. Do we care? I'd have to thread `fetchCartId` into this action if we want to leave it.